src/Illuminate/Log/Context/Repository.php

```diff
@@ -369,6 +369,35 @@ public function popHidden($key)
         return array_pop($this->hidden[$key]);
     }
 
+    /**
+     * Increment a context counter.
+     *
+     * @param  string  $key
+     * @param  int  $amount
+     * @return $this
+     */
+    public function increment(string $key, int $amount = 1)
+    {
+        $this->add(
+            $key,
+            (int) $this->get($key, 0) + $amount,
+        );
+
+        return $this;
+    }
+
+    /**
+     * Decrement a context counter.
+     *
+     * @param  string  $key
+     * @param  int  $amount
+     * @return $this
+     */
+    public function decrement(string $key, int $amount = 1)
+    {
+        return $this->increment($key, $amount * -1);
```

Comment:
Would `-$amount` be the same as `$amount * -1`? And is it any better performance-wise?

Reply:
Yes, but matching the way it's done elsewhere in the framework. Lots of examples, but e.g. https://github.com/laravel/framework/blob/e6753fc3836bb5fdddd1ee95835d8065bd7c5b16/src/Illuminate/Session/Store.php#L450